manual/src/full---help-output.md

```diff
@@ -632,165 +855,234 @@ Options:
       --24-bit-color <auto|always|never>
           Deprecated: use --true-color
 
+          [possible values: auto, always, never]
+
   -h, --help
           Print help (see a summary with '-h')
 
   -V, --version
           Print version
 
-GIT CONFIG
-----------
 
-By default, delta takes settings from a section named "delta" in git config files, if one is present. The git config file to use for delta options will usually be ~/.gitconfig, but delta follows the rules given in https://git-scm.com/docs/git-config#FILES. Most delta options can be given in a git config file, using the usual option names but without the initial '--'. An example is
+Git config
+
+  By default, delta takes settings from a section named "delta" in git config
```

Comment:
This mentions the git config, but should also mention how to set `pager = delta`. This way `--help` is a stand-alone manual, at least for the basics.

And this quick start section should be right after `Arguments:`, but that will require some more `--help` surgery.

Reply:
> This mentions the git config, but should also mention how to set pager = delta. This way --help is a stand-alone manual, at least for the basics.

Agreed. Very happy for you to add this but let me know if you want me to do anything.

Reply:
This PR is long enough, lets improve the help content in a different PR :)